tools/dependency_summary.py

```diff
@@ -924,6 +931,7 @@ def _find_license_file(self, id, license, pkgInfo):
             licenseFileNames = COMMON_LICENSE_FILE_NAMES[license]
         except KeyError:
             licenseFileNames = COMMON_LICENSE_FILE_NAMES[""]
+        logging.debug("candidate license filenames: %s", licenseFileNames)
         foundLicenseFiles = [nm for nm in os.listdir(
             pkgRoot) if nm.lower() in licenseFileNames]
```

Comment:
See e.g. here where we use `nm.lower()`.

Reply:
I've now removed them, and if the issue comes up again, we can debug the root cause, rather than papering over.